components/collections/src/codepointtrie/builder/Cargo.toml

```diff
@@ -24,7 +24,7 @@ include = [
 ]
 
 [dependencies]
-icu_codepointtrie = { path = "..", features = ["serde"] }
+icu_collections = { path = "../../..", features = ["serde"] }
```

Comment:
nit: this crate should probably not live under a src/ folder, prefer `collections/codepointtrie_builder` instead

Reply:
Agreed. Will do subsequently in a followup PR.